backoffice/src/Models/Domain/Droit.cs

```diff
@@ -1,7 +1,59 @@
-namespace Droits.Models
+namespace Droits.Models;
+
+public class Droit
 {
-    public class Droit
-    {
-       public Guid Id{get;set;}
-    }
-}
+    // Base fields...
+
+    public Guid Id { get; set; }
+
+    public string? Reference { get; set; } // This is the current reference.
+    public DroitStatus Status { get; set; } = DroitStatus.Received;
+    public DateTime ReportedDate { get; set; } = DateTime.UtcNow;
+    public DateTime Created { get; set; } = DateTime.UtcNow;
+    public DateTime Modified { get; set; } = DateTime.UtcNow;
+
+
+    // public List<DroitItem> WreckMaterials {get;set;} = new List<DroitItem>();
+
+    // Wreck
+
+    public Guid? WreckId { get; set; }
+
+    public bool IsHazardousFind { get; set; } = false;
+
+    // Wreck Vessel
+
+    public string? WreckVesselName { get; set; } = string.Empty;
+    public string? WreckConstructionDetails { get; set; } = string.Empty;
+    public int? WreckVesselYearConstructed { get; set; }
+    public int? WreckVesselYearSunk { get; set; }
+
+
+    // Location
+    public string? Latitude { get; set; }
+    public string? Longitude { get; set; }
+    public bool InUkWaters { get; set; } = false;
+    public int? LocationRadius { get; set; } //Units? Unknown.
+    public int? Depth { get; set; } //Units? Unknown. //metres from frontend
+    public string? LocationDescription { get; set; } = string.Empty;
```

Comment:
Is there are reason why the domain models look more like data structures than objects? Are we going to flesh them out with more behaviour as we learn more about the domain so they don't stay so anaemic?

Reply:
Yeah as they're unknowns ATM, makes sense to keep them as basic as possible. In theory these will probably stay quite lightweight as the entity framework models, then we'd expand them with functional methods, or have extension methods to keep them off the core model